### PR TITLE
[1.5.0] Updated CLI doc for `apply` command result

### PR DESCRIPTION
**Pull request by NoSkillGirl**
Signed-off-by: NoSkillGirl <singhpooja240393@gmail.com>

Related issue - https://github.com/kyverno/website/issues/281

### CODE REVIEW

File: content/en/docs/Kyverno CLI/_index.md [hunk 1]
```diff
@@ -216,6 +216,15 @@ policies:
 kyverno apply /path/to/add_network_policy.yaml --resource /path/to/required_default_network_policy.yaml -f /path/to/value.yaml
 ```
 
+On applying the above policy to the mentioned resources, the following output will be generated:
+```sh
```
Comment (chipzoller) [inline content/en/docs/Kyverno CLI/_index.md:220]: Very minor thing, but in order to pass Markdown linting tests, fenced code blocks should be surrounded by a single new line.

File: content/en/docs/Kyverno CLI/_index.md [hunk 2]
```diff
@@ -216,6 +216,15 @@ policies:
...
+Applying 1 policy to 1 resource... 
+(Total number of result count may vary as the policy is mutated by Kyverno. To check the mutated policy please try with log level 5)
+
+pass: 1, fail: 0, warn: 0, error: 0, skip: 0 
+```
```
Comment (chipzoller) [inline content/en/docs/Kyverno CLI/_index.md:225]: Ditto

**Review by chipzoller — APPROVED** [review]
Looks good, just minor line spacing to make linters happy.

**Comment by NoSkillGirl** [comment]
@chipzoller - I do not have permission to merge the PR. Please check and merge the changes.

**Pull request merged**